uaclient/cli.py

```diff
@@ -779,6 +847,9 @@ def wrapper(*args, **kwargs):
             with util.disable_log_to_console():
                 logging.exception("KeyboardInterrupt")
             print("Interrupt received; exiting.", file=sys.stderr)
+            if _LOCK_FILE and os.path.exists(_LOCK_FILE):
```

Comment:
Since we are using the same code in three different places in this function, maybe we could move to the `util` module as well